loops-arrays-objects.js

```diff
@@ -181,3 +182,4 @@ for (let i = 0; i < students.length; i++) {
 }
 console.log(students);
 
```

Comment:
In de uitwerkingen willen ze een switch-statement van je, maar ik heb het op dezelfde manier gedaan en het scheelt heel veel code t.o.v. de uitwerkingen :) En het is makkelijker om een wijk toe te voegen. Dus top! 
1 kritische noot: Als ik een zipcode invul die niet gemapt is (of een typfout maak), dan overschrijft jouw loop de null-waarde naar undefined. als je || null toevoegt aan het einde (dus neighborhoodMapping[zipcode] || null), dan laat hij het op null staan als de zipcode niet in de Map-variabele staat.